openid-sharedsignals-framework-1_0.md

```diff
@@ -788,7 +788,7 @@ following properties:
 stream_id
 
 > **Transmitter-Supplied**, A string that uniquely identifies the stream. Stream
-  IDs MUST be unique per Reciever.  This value is generated by the Transmitter
+  IDs MUST be unique per stream in the context of each transmitter.  This value is generated by the Transmitter
```

Comment:
"across all streams that are currently open in a Transmitter"

Reply:
Maybe the following to handle "future" streams, rather than just the "currently open"

> **Transmitter-Supplied**, A string that uniquely identifies the stream. A transmitter MUST generate a unique ID for each of it's streams.